api/Window.json

```diff
@@ -2746,6 +2746,48 @@
           }
         }
       },
+      "lostpointercapture_event": {
```

Comment:
Are you sure the event can target a Window as opposed to bubbling up from a Document?